Sources/SwerverCore/Request/RequestMethod.swift

```diff
@@ -1,4 +1,5 @@
-public enum RequestMethods: String {
+public enum RequestMethod: String {
     case get = "GET"
+    case head = "HEAD"
     case other = "OTHER"
```

Comment:
Why do you have an "OTHER" verb?

Reply:
I used it to indicate when the request is an unknown type. I'll remove it since the router is returning an empty response now